modules/custom/openy_upgrade_tool/src/OpenyUpgradeLogListBuilder.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace Drupal\openy_upgrade_tool;
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Entity\EntityListBuilder;
+use Drupal\Core\Link;
+use Drupal\Core\Url;
+
+/**
+ * Defines a class to build a listing of Openy upgrade log entities.
+ *
+ * @ingroup openy_upgrade_tool
+ */
+class OpenyUpgradeLogListBuilder extends EntityListBuilder {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildHeader() {
+    $header['name'] = $this->t('Config Name');
+    $header['updated'] = $this->t('Updated');
+    $header['user'] = $this->t('By User');
+    $header['status'] = $this->t('Status');
+    return $header + parent::buildHeader();
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildRow(EntityInterface $entity) {
+    $row['name'] = Link::createFromRoute(
+      $entity->label(),
+      'entity.openy_upgrade_log.canonical',
+      ['openy_upgrade_log' => $entity->id()]
+    );
+    $row['updated'] = \Drupal::service('date.formatter')->format($entity->getChangedTime());
```

Comment:
let's use DI, please

Reply:
I will remove this columns and leave only the name.
List builder used as a dashboard in the beginning, for now we need only `getOperations` from this class.